Gemfile

```diff
@@ -66,10 +66,12 @@ end
 gem 'devise'
 
 group :test do
+  gem "faker", git: "https://github.com/stympy/faker.git", branch: "master"
   gem 'shoulda-matchers'
   gem 'rails-controller-testing'
 end
 
 gem 'bootstrap', '~> 4.3.1'
 
 gem 'rubocop', '~> 0.68.1', require: false
```

Comment:
Move this 2 gems on top of the file, delete all comments in here and add all the gems that the practice is requiring